### PR TITLE
Fixes for compile-and-run.py

### PR DESCRIPTION
**Pull request by tjlaboss**
Found this file while grepping for something else. I'm not sure what it is, but it hasn't been updated with the rest of the code.

* Changed `print` to `print()`
* Indented with 4 spaces instead of 2
* Replaced hardcoded `"geogunow"` with `os.environ['USER']` for users not named geogunow

### CODE REVIEW

File: profile/results/compile-and-run.py [hunk 1]
```diff
@@ -17,142 +17,143 @@
...
-  x = set()
-  for i in range(1, min(num, max_decomp) + 1):
-    for j in range(1, min(num/i, max_decomp) + 1):
-      for k in range(1, min(num/(i*j), max_decomp) + 1):
-        if (i*j*k == num):
-          x.add((i,j,k))
-  return list(x)
+    x = set()
+    for i in range(1, min(num, max_decomp) + 1):
+        for j in range(1, min(num/i, max_decomp) + 1):
```
[the remark this answers is not in the document]
Comment (tjlaboss) [reply profile/results/compile-and-run.py]: Will do floor division, `num//i`

File: profile/results/compile-and-run.py [hunk 2]
```diff
@@ -17,142 +17,143 @@
...
+                        words = line.split()
+                        words = words[0].split('->')
+                        words = words[0].split('.')
+                        geometry = words[0]
+                        pieces = line.split(geometry)
+                        white_space = pieces[0]
+                        breaker = pieces[1].split('setRootUniverse')[0]
+                        write_line = white_space + geometry + breaker + \
+                                'setDomainDecomposition(' + \
+                                str(d[0]) + ', ' + str(d[1]) + ', ' + str(d[2]) + ');\n'
```
[the remark this answers is not in the document]
Comment (tjlaboss) [reply profile/results/compile-and-run.py]: Not sure what that argument should look like. Is it literally just `MPI_COMM_WORLD`?

[the remark this answers is not in the document]
Comment (GiudGiud) [reply profile/results/compile-and-run.py]: yup

**Comment by tjlaboss** [comment]
Done, how does that look?

**Comment by GiudGiud** [comment]
Looks good. Merging !

**Pull request merged**